packages/test-base/src/commonTest/kotlin/io/realm/kotlin/test/common/RealmSchemaTests.kt

```diff
@@ -95,18 +97,19 @@ class RealmSchemaTests {
             ?: fail("Couldn't find class")
         assertEquals(SCHEMA_VARIATION_CLASS_NAME, schemaVariationsDescriptor.name)
         assertFalse(schemaVariationsDescriptor.isEmbedded)
+        assertEquals(schemaVariationsDescriptor.kind, RealmClassKind.STANDARD)
         assertEquals("string", schemaVariationsDescriptor.primaryKey?.name)
 
         val sampleName = "Sample"
         val sampleDescriptor = schema[sampleName] ?: fail("Couldn't find class")
         assertEquals(sampleName, sampleDescriptor.name)
-        assertFalse(sampleDescriptor.isEmbedded)
+        assertNotEquals(sampleDescriptor.kind, RealmClassKind.EMBEDDED)
```

Comment:
I still think we need to test both `isEmbedded` and `kind` until we remove `isEmbedded`